docs/gpa.md

```diff
@@ -150,7 +152,10 @@ This requires the [Additional Contact Points](https://github.com/open-contractin
         <td>a list and brief description of any conditions for participation of suppliers, including any requirements for specific documents or certifications to be provided by suppliers in connection therewith, unless such requirements are included in tender documentation that is made available to all interested suppliers at the same time as the notice of intended procurement;</td>
         <td markdown=1>
 
-* Add this to `tender/eligibilityCriteria`
+* Add *a list and brief description of any conditions for participation of suppliers* to `tender/eligibilityCriteria`
+* If *requirements are included in tender documentation that is made available to all interested suppliers*:
+  * For each document, add a `Document` object to the `tender/documents` array
+  * Set its `documentType` to 'eligibilityCriteria'
```

Comment:
I think this clause refers to selection criteria, similar to those in TED forms: https://github.com/open-contracting/standard/issues/901 As I remember, TED forms had a few different ways of expressing such criteria.

Reply:
https://github.com/open-contracting-extensions/government-procurement-agreement/issues/30